host/xtest/CMakeLists.txt

```diff
@@ -88,6 +88,7 @@ target_include_directories(${PROJECT_NAME}
 	PRIVATE adbg/include
 	PRIVATE xml/include
 	PRIVATE ${OPTEE_TEST_SDK}/host_include
+	PRIVATE ${CMAKE_CURRENT_BINARY_DIR}
```

Comment:
Why this change?

Reply:
Because of this parameter:

`--out ${CMAKE_CURRENT_BINARY_DIR}/regression_8100_${prefix}.h`

for `file_to_c.py` script.

I think, it should be in sync. 

Also, it is not building without this change.

Reply:
ok, thanks.